### ISSUE TITLE
pre-trainning model

### ISSUE THREAD

**Issue by Gaoang1** [huangleiBuaa/XBNBlock#1]
Can you provide your pre-training model for Resnet?

**Comment by huangleiBuaa** [comment]
@Gaoang1 , you can find the pretrained resnet-50 in here (starndard train):  https://drive.google.com/file/d/1xqGsCDD4Y_rv6PjTtACAs8D918fjUT55/view?usp=sharing 

 and here (advanced train): 
https://drive.google.com/file/d/1pHRhjc67M5wki5fWy44SG5lj06q71gat/view?usp=sharing

**Comment by Gaoang1** [comment]
> @Gaoang1 , you can find the pretrained resnet-50 in here (starndard train): https://drive.google.com/file/d/1xqGsCDD4Y_rv6PjTtACAs8D918fjUT55/view?usp=sharing
> 
> and here (advanced train): https://drive.google.com/file/d/1pHRhjc67M5wki5fWy44SG5lj06q71gat/view?usp=sharing

Thank you, and what is the difference between standard and advanced?

**Comment by Gaoang1** [comment]
> > @Gaoang1 , you can find the pretrained resnet-50 in here (starndard train): https://drive.google.com/file/d/1xqGsCDD4Y_rv6PjTtACAs8D918fjUT55/view?usp=sharing
> > and here (advanced train): https://drive.google.com/file/d/1pHRhjc67M5wki5fWy44SG5lj06q71gat/view?usp=sharing
> 
> Thank you, and what is the difference between standard and advanced?

Thank you

**Comment by huangleiBuaa** [comment]
@Gaoang1 , standard training using the standard 100 epoch with step decay, advanced training using 100 epoch with cosine decay, combining label smooth and MixUP. Please see the paper for detail.

**Comment by Gaoang1** [comment]
thank you！




------------------&nbsp;原始邮件&nbsp;------------------
发件人: "Lei ***@***.***&gt;; 
发送时间: 2022年4月24日(星期天) 上午9:16
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [huangleiBuaa/XBNBlock] pre-trainning model (Issue #1)





 
@Gaoang1 , standard training using the standard 100 epoch with step decay, advanced training using 100 epoch with cosine decay, combining label smooth and MixUP. Please see the paper for detail.

**Comment by Gaoang1** [comment]
Hi,I also would like to your the pretrained resnext_p2 ,Could you please provide it ？Thanks！

**Comment by huangleiBuaa** [comment]
> 

See the pre-trained model of ResNeXt-50-XBNBlock (advanced_train): https://drive.google.com/file/d/1MKQ4iAOnKGWmDwN7lfzFJqGmTqL823NJ/view?usp=sharing
